clang/lib/DPCT/PatternRewriter.cpp

```diff
@@ -388,6 +393,33 @@ static bool isIdentifiedChar(char Char) {
   return false;
 }
 
+static void
+updateExtentionName(const std::string &Input, size_t Next,
+                    std::unordered_map<std::string, std::string> &Bindings) {
+  if (Input.compare(Next, strlen(".cpp"), ".cpp") == 0) {
```

Comment:
what if .cxx  .C , *c?

Reply:
Will implement this feature in a separate PR.